bundle/edu.gemini.shared.skyobject/src/main/java/edu/gemini/shared/skyobject/Magnitude.java

```diff
@@ -45,6 +48,18 @@
         AP(None.INTEGER, new Some<>("apparent"))
         ;
 
+        public static final List<Band> AB_BANDS;
+
+        static {
+            List<Band> bands = new ArrayList<>();
+            bands.add(u);
+            bands.add(g);
+            bands.add(r);
+            bands.add(i);
+            bands.add(z);
+            AB_BANDS = Collections.unmodifiableList(bands);
+        }
```

Comment:
What about putting the default system as a field of the band itself?  That would force you to think about it whenever you add a new band.